packages/cli/__tests__/zosfiles/__system__/create/ds/__scripts__/command/command_create_ds_like.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+echo "================Z/OS FILES CREATE DS==============="
+zowe files create data-set my.data.copy --like my.data.orig 
```

Comment:
Please parameterize this line so the system test can pass values into the script. Something like this should work:
`zowe files create data-set "$1.test.data.set.like" --like $2`

For a similar example, see `packages/cli/__tests__/zosfiles/__system__/create/binaryPds/__scripts__/command/command_create_binary_pds.sh`